projects/hathor-wallet-mobile/security/0001-improved-pin-security-mode.md

```diff
@@ -0,0 +1,66 @@
+# Summary
+[summary]: #summary
+
+A new way to use biometrics on the mobile wallet, using a longer pin for security, this does not allow a fallback to the configured pin in case of too many failures.
+
+# Motivation
+[motivation]: #motivation
+
+When using biometrics the pin is saved on the phone system, meaning the storage data is still encrypted with the 6 digit pin, using a longer and more random pin makes the storage data harder to crack.

```

Comment:
This makes me wonder if we should also review the security mode for the pin itself. But I believe it is for "future possibility".